docs/index.html

```diff
@@ -42,15 +43,69 @@ <h1><code>wasm-pack</code></h1>
           <h2>📦✨ your favorite rust -> wasm workflow tool!</h2>
         </div>
         <div class="five columns" id="installer">
-          <a class="button button-primary" href="/wasm-pack/installer">✨ Install wasm-pack 0.5.1 ✨</a>
+          <a class="button button-primary" href="/wasm-pack/installer">✨ Install✨</a>
           <p>9 Oct 2018 |
             <a href="https://github.com/rustwasm/wasm-pack/releases/tag/v0.5.1">
-              Release Notes
+              0.5.1 Release Notes
             </a>
           </p>
         </div>
       </div>
     </header>
+    <hr/>
+    <section id="build">
+      <div class="container">
+        <h2>Build it with wasm-pack!</h2>
+        <div class="row">
+          <div class="four columns">
+            <header>
+              <img src="./public/img/webpack.jpeg" alt="webpack" />
+              <h4>Hybrid Applications with Webpack</h4>
+            </header>
+            <p>
+              Develop a WebAssembly module alongside any JavaScript project
+              that leverages <a href="https://webpack.js.org/">Webpack</a>.
+              Edit your Rust and watch your WebAssembly compile and reload as
+              you work- a great way to start exploring WebAssembly in a JavaScript
+              application you already work on!
+            </p>
+            <a href="#" class="button">Learn More</a>
+          </div>
+          <div class="four columns">
+            <header>
+              <img src="./public/img/npm-packages.png" alt="npm packages" />
+              <h4>npm Packages for the Broswer and Node.js</h4>
```

Comment:
Typo, `Broswer` -> `Browser`. Also, should it be capitalized?

Reply:
good catch! and no probably not- will fix